pkg/ast/spl/tests/splParser_test.go

```diff
@@ -7793,6 +7795,306 @@ func Test_Format_cmd_Incomplete_RowCol_Options(t *testing.T) {
 	assert.NotNil(t, err)
 }
 
+func getTimeDiff(daylight bool) int64 {
+	localTime := time.Date(2024, time.June, 5, 13, 37, 5, 0, time.Local)
+	if daylight {
+		localTime = time.Date(2024, time.January, 1, 0, 0, 0, 0, time.Local)
+	}
```

Comment:
Isn't daylight saving time in the summer?

Reply:
removed this to use fixed dates for clean code. Addressed in the new commit.